tools/tertiary-analysis/decoupler/test-data/progeny_test.tsv

```diff
@@ -0,0 +1,71 @@
+	source	target	weight	p_value
```

Comment:
is it required to have an index column? might be easier for users and workflows that the script doesn't expect an index column? Or if not provided it works as well?

Reply:
I havent tried without index but thats how it is in docs

Reply:
It works with and without index, changes in [3f4efd6](https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/pull/308/commits/3f4efd648085c588e5633396c2f0a1088f93b3a3)

Reply:
test added [2d76c11](https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/pull/308/commits/2d76c1111cedda721943056de7b4fa3da1c238fe)